src/Slider/LayerTransparencySlider.example.jsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import { render } from 'react-dom';
+
+import OlMap from 'ol/map';
+import OlView from 'ol/view';
+import OlTileLayer from 'ol/layer/tile';
+import OlOsmSource from 'ol/source/osm';
```

Comment:
Plz use import names as declared in #58 
so `OlTileLayer` would become `OlLayerTile`

Reply:
Thanks for the hint!